### ISSUE TITLE
[Chip][Accessibility] Chip is not focusable by touch navigation with Talkback on API version 24

### ISSUE THREAD

**Issue by caleb-allen** [material-components/material-components-android#2155]
**Description:** On API level 24 with Talkback enabled, a user is unable to navigate Accessibility focus to a chip by touch exploration (either by single-tapping the view, or by dragging their finger to explore the views). A user is still able to navigate to a chip by swiping through the view tree. Videos of bug below.

**Expected behavior:** A user is able to move Accessibility focus to a chip via single-tap or touch exploration.

**Source code:** Material-Components Catalog App

**Android API version:** 24

**Material Library version:** 1.2.1, 1.3.0, 1.4.0-alpha01

**Device:** Alcatel Verso, Android Emulator

Here is the bug in our app:

https://user-images.githubusercontent.com/3279748/112698464-f4274000-8e46-11eb-8c9e-4f020c3caf9c.mp4

Here is a video of the bug in the Material Components Catalog App, built from source, running on the emulator on API level 24. In our testing this is the only API level where we have encountered the bug.

https://user-images.githubusercontent.com/3279748/112698702-7ca5e080-8e47-11eb-8db1-bb3b9b2b9923.mp4

**Comment by caleb-allen** [comment]
My guess is that the `ChipTouchHelper` is interfering with touch events, even when no close icon is provided
